Tweak.xm

```diff
@@ -267,6 +275,14 @@ BOOL override = NO;
     return value;
 }
 
+- (BOOL)isPictureInPicturePossible {
+    forceEnablePictureInPictureInternal([self valueForKey:@"_hotConfig"]);
+    override = YES;
+    BOOL value = %orig;
+    override = NO;
+    return value;
+}
+
```

Comment:
I don't see the need to hook this method because the method that it calls already gets hooked.
![image](https://user-images.githubusercontent.com/3608783/123531810-ddd49d00-d731-11eb-8195-3a59a488a197.png)